crates/ruff_cli/src/diagnostics.rs

```diff
@@ -254,10 +254,9 @@ pub(crate) fn lint_path(
                                 // mutated it.
                                 let src_notebook = source_kind.as_ipy_notebook().unwrap();
                                 let mut stdout = io::stdout().lock();
-                                for ((idx, src_cell), dest_cell) in src_notebook
-                                    .cells()
-                                    .iter()
-                                    .enumerate()
+                                // Cell indices are 1-based.
+                                for ((idx, src_cell), dest_cell) in (1u32..)
+                                    .zip(src_notebook.cells().iter())
                                     .zip(dest_notebook.cells().iter())
```

Comment:
Should we use a type like `OneIndexed` to make it clear in the API which indices are one or zero indexed?

Reply:
Yes, I'm planning on making that refactor for `NotebookIndex` which will be a separate PR.